receiver/podmanreceiver/receiver.go

```diff
@@ -87,3 +168,56 @@ func (r *receiver) scrape(context.Context) (pdata.Metrics, error) {
 	}
 	return md, nil
 }
+
+func (r *receiver) handleEvents(ctx context.Context, eventBackoff *backoff.ExponentialBackOff) error {
+	c, err := r.clientFactory(r.set.Logger, r.config)
+	if err != nil {
+		r.set.Logger.Error("error fetching/processing events", zap.Error(err))
+		return err
```

Comment:
I think it's better to log where the error is finally handled/relevant instead of at each passed point:

```suggestion
		return fmt.Errorf("error fetching/processing events: %w", err)
```

There are several places where redundant logging is done.

Reply:
Done